### DISCUSSION TITLE
signalk plugins installed list

### DISCUSSION THREAD

**Discussion by gildas-declercq** [bareboat-necessities/lysmarine_gen#452] [Q&A]
Hi all,
how can I know the signalk plugin installed in a given version of BBN OS ?

thanks a lot
Gildas

**Comment by mgrouch** [comment]
From SignalK store web UI

**Comment by gildas-declercq** [comment]
yes I did that,
but I would like to have it before installing

**Reply by mgrouch** [reply]
You can specify version you want to install explicitly with @  in install script
